neuron_morphology/pipeline/post_data_to_s3.py

```diff
@@ -0,0 +1,101 @@
+import os
+import boto3
+import json
+from zipfile import ZipFile, ZIP_DEFLATED
+from io import BytesIO
+from argschema import ArgSchemaParser
+from neuron_morphology.pipeline._schemas import InputParameters
+
+def zip_files(file_list):
+    """
+    zip files into an archive in memory
+
+    Parameters
+    ---------------
+    file_list: file paths or file in bytes to be archived
```

Comment:
file_list is a dict, please describe what are the keys and what are the values

Reply:
revised